vector-stores/spring-ai-milvus-store/src/main/java/org/springframework/ai/vectorstore/MilvusVectorStore.java

```diff
@@ -69,6 +69,8 @@ public class MilvusVectorStore implements VectorStore, InitializingBean {
 
```

Comment:
Please add your name as an author to the class.

Reply:
Thanks!